pyflask/manageNeuroconv/manage_neuroconv.py

```diff
@@ -72,6 +77,65 @@ def get_metadata_schema(source_data: Dict[str, dict]) -> Dict[str, dict]:
     return json.loads(json.dumps(dict(results=metadata, schema=schema), cls=NWBMetaDataEncoder))
 
 
+def get_check_function(check_function_name: str) -> callable:
+    """
+    Function used to fetch an arbitrary NWB Inspector function
+    """
+    check_function: callable = nwbinspector.__dict__.get(check_function_name)
+    if check_function is None:
+        raise ValueError(f"Function {function} not found in nwbinspector")
+
+    return check_function
+
+
+def validate_subject_metadata(subject_metadata: dict, check_function_info: str or callable):
```

Comment:
```suggestion
def validate_subject_metadata(subject_metadata: dict, check_function_info: str or callable) -> Union[None, InspectorMessage, List[InspectorMessage]]:
```

And ditto for the other validate helper

Note will have to import like `from nwbinspector.register_checks import InspectorMessage`